cocos/core/animation/skeletal-animation-blending.ts

```diff
@@ -165,6 +185,7 @@ interface NodeBlendState {
     properties: {
         position?: PropertyBlendState<Vec3>;
         rotation?: PropertyBlendState<Quat>;
+        eulerAngles?: PropertyBlendState<Vec3>;
         scale?: PropertyBlendState<Vec3>;
```

Comment:
@YunHsiao  Please note that only curves that operating  on these properties are subjected to skeletal blending(eg. calling `setTRS`).

Reply:
That's okay for now